website/src/supportedNetworks.tsx

```diff
@@ -75,16 +79,15 @@ export function SupportedNetworksTable({ networks }: { networks: Awaited<ReturnT
             <td>
               <CodeInline>{network.cliName}</CodeInline>
             </td>
-            <td align="center">{network.chainId}</td>
+            <td align="center">{network.integrationType}</td>
             <td align="center">{network.supportedOnHostedService ? '✓' : null}</td>
             <td align="center">
               {network.supportedOnStudio ? '✓' : null}
               {network.substreams.includes('studio') ? <sup>†</sup> : null}
             </td>
             <td align="center">
               {network.fullySupportedOnNetwork ? '✓' : null}
-              {network.partiallySupportedOnNetwork ? '*' : null}
```

Comment:
Looks like you removed the only use of `partiallySupportedOnNetwork`, should we remove it from above?

Reply:
Makes sense to me, removed